src/de/jost_net/JVerein/server/FormularImpl.java

```diff
@@ -158,4 +158,98 @@ public Object getAttribute(String fieldName) throws RemoteException
     return super.getAttribute(fieldName);
   }
 
+  @Override
+  public int getZaehler() throws RemoteException
+  {
+    Integer counter = (Integer) getAttribute("zaehler");
+    if (counter == null)
+    {
+      return 0;
+    }
+    return (int) getAttribute("zaehler");
+  }
+
+  @Override
+  public void setZaehler(int zaehler) throws RemoteException
+  {
+    setAttribute("zaehler", zaehler);
+  }
+
+  @Override
+  public void setZaehlerToFormLink(int zaehler) throws RemoteException
+  {
+    DBIterator<Formular> formList = getLinked();
+    while (formList.hasNext())
+    {
+      Formular form = formList.next();
+      form.setZaehler(zaehler);
+      try
+      {
+        form.store();
+      }
+      catch (RemoteException e)
+      {
+        String fehler = "Fortlaufende Nr. kann nicht gespeichert werden. Siehe system log.";
+        Logger.error(fehler, e);
+        throw new RemoteException(fehler);
+      }
+      catch (ApplicationException e)
```

Comment:
"ApplicationException" sollten nicht gefangen werden. Die enthalten bereits sprechende Fehlermeldungen und können nach oben geworfen werden. Jameica kümmert sich darum, die Meldung als sauber formatierte Fehlermeldung in der Benutzeroberfläche anzuzeugen.